unlock-app/src/components/creator/members/GrantKeysDrawer.tsx

```diff
@@ -252,23 +249,19 @@ const GrantKeyForm = ({ onGranted, lock }: GrantKeyFormProps) => {
             })}
           />
           {errors.recipient && (
-            <p className="text-xs -mt-4 text-[#f24c15]">
+            <p className="text-xs text-[#f24c15]">
               Please make sure you enter a valid Ethereum address
             </p>
           )}
         </div>
```

Comment:
Let's add a little explanation, similar to the one we have for the key manager:
"If you enter the recipient's email address, they will receive an email to confirm the airdrop, as well as a link to view their NFT. The email will also include a QR code of the signed NFT that they can use to prove that they own it in when attending in person events."

Reply:
for email a

> This is looking very promising.

Updated with small changes in comments!